cadasta/test/util.py

```diff
@@ -20,5 +20,5 @@ def random_string():
 
 
 def random_us_number():
-    rand_digits = [str(random.randint(0, 9)) for i in range(7)]
+    rand_digits = [str(random.randint(1, 9)) for i in range(7)]
```

Comment:
This can result in numbers like "+16281258765", which also not valid. Changing this to 

```
rand_digits = [str(random.randint(2, 9)) for i in range(7)]
```